### ISSUE TITLE
[#SomeChanges Suggestion] Hit Rating interaction should be corrected to further improve the PVP gameplay

### ISSUE THREAD

**Issue by Evolvee** [JamminL/wotlk-classic-bugs#143]
### Description

**The issue is described pretty much already in here:**
 https://eu.forums.blizzard.com/en/wow/t/lf-a-blizzard-response-all-talents-that-reduce-spell-resists-in-pvp-no-longer-works-since-phase-2/320188

**The Blizzard response to this topic in here:**

https://tbc.wowhead.com/news/blizzard-explains-bug-fix-regarding-spell-resist-in-burning-crusade-classic-324594

**TLDR**: Excess Spell hit rating above the general PVP hit cap and all talents improving either chance to hit or reducing the chance to resist should further negate all talents/spell effects that increase a chance to resist (miss) such spell. In TBC you can pick any class and find such talent, in WOTLK these talents were either converted to flat reduction of CC duration or reworked entirely. The intention was clearly to move from random resists on key CC abilities and thats also what this topic proposes.

The fun part is that this is how melee hit rating already works _(classic tbc)_... unfortunately, spell hit _(and all classes relying on it, namely shadow priest)_ got intentionally nerfed in season 2 to the point of being useless (other than providing 2.66 spell pene to binary spells for each % of overcap) for no reason other than introducing even more annoying rng to an already rng-infested game.

I dont have the beta access but I am pretty sure this is how the mechanic currently works. 

The proposition is to revert the functionality of this mechanic back how it worked during classic TBC season 1. This will greatly improve the pvp gameplay by reducing the amount of annoying CC resists due to racials (e.g. undead shadow resist) and/or spell effects such as Sacred Cleansing.

**Examples of spell effects that this proposition would beneficially affect:** 
Rogue - Heightened Senses
Paladin - Sacred Cleansing
Paladin - Divine Purpose
Undead - Shadow Resistance (entry 20579)

**Pros:**
- Much more predictable gameplay.
- Players get the ability to atleast partially improve their overal resist/miss chance instead of being hopeless and forced to lose games just because of bad dev decisions that they cannot negate in any possible way.
- The game makes much more sense, spells and items are doing exactly what they say they should do.
- Improves the competitive part of PVP
- Rewards players who are "clever" about their gear choices.

**Cons:**
- Nada

**Comment by Jgrenier91** [comment]
I would love to see +Hit work above the hit cap and I approve of this.

Example: If a rogue decides to spec into Heightened Senses (reduces the chance you are hit by spells and ranged
attacks by 2%.) and play as Blood elf, a player with 0 hit would have a 10% chance to miss the rogue in PVP (4% Base + 2% Blood Elf + 4% talent). 
I think it's more fun to be able to choose to collect more hit rating/talents to negate some of these talents, especially since gathering anything over the 4% base hit rating will only have an affect vs specific races/classes so, there is a diminishing return on gathering more hit beyond 4%. Having choice is good and interesting in this case!

**Comment by Wallirik** [comment]
I also think this is a great suggestion. I can see no downsides as to why you shouldn't be able to cap -spell hit from talents, when physical abilities allow you to cap the same type of talents.

If you choose to cap through all of it (a staggering 10% hit if you want to cover racials+talents) you sacrifice some overall stats for consistency. There is a trade-off to capping this much hit, and I think that's a good thing.

**Comment by Loregon** [comment]
World of Warcraft Client Patch 3.0.8 (2009-01-20)
Racial Resistances: These can now be mitigated against by gaining additional chance to hit.
https://web.archive.org/web/20100807220839/http://www.worldofwarcraft.com/patchnotes/patch3p09.html

The mechanic could have changed, and after fixing the 1% chance of resist, it would probably be worth checking it out for those who have beta.

**Comment by JamminL** [comment]
> ### World of Warcraft Client Patch 3.0.8 (2009-01-20)
> Racial Resistances: These can now be mitigated against by gaining additional chance to hit. https://web.archive.org/web/20100807220839/http://www.worldofwarcraft.com/patchnotes/patch3p09.html
> 
> The mechanic could have changed, and after fixing the 1% chance of resist, it would probably be worth checking it out for those who have beta.

This would indeed apply to talents as well. I'm closing this #SomeChanges. We'll need to test on beta if this is working like it should work

**Comment by Evolvee** [comment]
So aparently ppl on the beta tested it with extreme spell-hit overcap and it still resisted _(missed since miss and resist isnt distinguished, yet)._

**Comment by Dverikk** [comment]
If they don't wanna fix it because of game play, at least think about the mental health of losing a lot of games because off rng.

**Comment by Evolvee** [comment]
> > ### World of Warcraft Client Patch 3.0.8 (2009-01-20)
> > Racial Resistances: These can now be mitigated against by gaining additional chance to hit. https://web.archive.org/web/20100807220839/http://www.worldofwarcraft.com/patchnotes/patch3p09.html
> > 
> > The mechanic could have changed, and after fixing the 1% chance of resist, it would probably be worth checking it out for those who have beta.
> 
> This would indeed apply to talents as well. I'm closing this #SomeChanges. We'll need to test on beta if this is working like it should work

Why is this issue still closed? It was tested on the beta realm and its still obviously bugged/not working the way this issue proposes for it to work.

Even if the suggestion is rejected _(because certain ppl aparently love pvp to be pointless -e.g. the fully intentional change made in TBCC season2 and onwards)_, the mechanic described in the patch note above simply doesnt work this topic should atleast be changed from "somechanges suggestion" into a regular "bug".

**Comment by Aggrend** [comment]
I commented on another issue about the removal of heartbeat resists in arena, and as part of that change, we've also restored the original behavior from original Wrath described above involving the ability to be mitigate additional resists via additional Spell Hit. To be clear, this means that in Wrath classic you will in fact be able to "overcap" hit to overcome additional resist chances gained from certain abilities and talents. This should be present in our new build on Beta and PTR tomorrow (8/10) and we'll include it in our build notes for the week when we do. 

Thanks!
